app/views/casa_admin_mailer/deactivation.html.erb

```diff
@@ -1,17 +1,45 @@
-<meta itemprop="name" content="Confirm Email" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
-<table width="100%" cellpadding="0" cellspacing="0" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
-  <tr style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
-    <td class="content-block" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; vertical-align: top; margin: 0; padding: 0 0 20px;" valign="top">
+<style>
```

Comment:
@efgalvao what do you think about moving this to a separated css file?

Reply:
I thought would be better, but because was asking for a <style> element I chose to let in the same file. What do you think @FireLemons ?

Reply:
it's not a good practice to have embedded CSS in the same template

Reply:
@brunohkbx  yeah the long term goal is to get a separate css bundle for the emails but it's hard to refactor with a lot of the css being inline

Reply:
@FireLemons oh, I didn't notice it was an email template 🤦🏻 , my bad!